app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,25 @@
+package ar.com.wolox.android.example.ui.home
+
+import ar.com.wolox.android.example.ui.home.News.Create.CreateNewsFragment
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+import javax.inject.Inject
+
+/**
+ * Home activity
+ */
+@Suppress("CAST_NEVER_SUCCEEDS")
+class HomeActivity @Inject constructor() : WolmoActivity() {
+
+    override fun layout(): Int = ar.com.wolox.android.R.layout.activity_base
+
+    override fun init() {
+        replaceFragment(ar.com.wolox.android.R.id.vActivityBaseContent, HomeFragment.newInstance(), "Home Fragment")
+    }
+
+    fun switchToCreateNewsFragment() {
```

Comment:
Y don't think that we need this. Remember that the ViewPager contains two instances of Fragments already